scripts/ts-generate-definition.js

```diff
@@ -33,12 +33,27 @@ fs.readFile(clientFile, 'utf8', (err, data) => {
         .replace(': ?', '?: ')
       ).join('\n');
 
+    const nullableTypes = types.split('\n\n')
+      .map(block => {
+        if (block.includes('Change = {')) {
```

Comment:
The added code transforms "undefined-able" properties into "nullable" properties. This change is desirable for all types returned by the API, because the API never returns omit any property (which would make them `undefined`). However, some types that are sent to the API have all their fields marked as "undefined-able". If these types underwent the transformation, the user would be required to fill them with "null", which is wrong.